src/main/java/com/example/application/ui/horizontal/projects/NordlichtView.java

```diff
@@ -25,16 +22,18 @@
 /**
  *  Nordlicht View shows ...
  *
- *  @author Monika Martius and Laura Neuendorf
- *  @version 2.0
+ *  @author Monika Martius and Laura Neuendorf, Jessica Reistel
+ *  @version 3.0
  *  @since 24.01.2020
- *  @lastUpdated 26.01.2020
+ *  @lastUpdated 01.02.2021 from Jessica Reistel, Monika Martius
  */
 @Route(value = "nordlicht", layout = MainView.class)
 @PageTitle("Nordlicht")
 public class NordlichtView extends Div {
 
     private ImagesManager imagesManager;
+    private ImagesManager imagesManagerTwo;
```

Comment:
There are 2 imageManager/imageUploadContainer, yet each time one image is uploaded, the image will be added twice in the containers.
![image](https://user-images.githubusercontent.com/47325924/106490588-463a8d80-64b6-11eb-829d-bc98dfd214d7.png)

Reply:
Das ist noch normal, weil es noch nicht angepasst wurde, das kommt aber noch in einem anderen Pull Request den @MoMa666 machen wollte

Reply:
Ja ihr habt das ja angepasst das es nun möglich ist zwei unterschiedliche Bilder einzustellen oder? Deswegen wollte ich es dann erst nach diesem Pull Request abändern. Hier geht es lediglich nur um die users und ihre Rollen nicht um die View selbst.

Reply:
Es wäre dann hilfreich eine Erklärung zu erhalten welchen konstruktor ich aufrufen muss damit das funktioniert oder einer von euch ändert das dann nach dem PullRequest ab

Reply:
kann das mal später ändern, geht eigentlich ohne einen neuen konstruktor

Reply:
die klasse wurde geändert, und die images Änderungen wurden in diesem pull request gepusht, deswegen haben wir das kommentiert

Reply:
Sollen wir hier jetzt noch irgendwas ändern oder kann das für den Request erstmal so bleiben?